qa/tasks/cephfs/test_admin.py

```diff
@@ -11,36 +11,33 @@
 from tasks.cephfs.caps_helper import CapsHelper
 
 
-class TestAdminCommands(CephFSTestCase):
+class AdminCmdTestCase(CephFSTestCase):
```

Comment:
I don't think this naming has any impact on whether this class is loaded, based on the discussion in this thread https://github.com/ceph/ceph/pull/41857#discussion_r652276366 .

I think the right thing to do here is multiple inheritance. This class should not inherit `CephFSTestCase`. Instead, child classes should inherit `TestAdminCommands` and `CephFSTestCase`.

Reply:
This PR is included in https://github.com/ceph/ceph/pull/41385
I will make the change in that bigger PR 41385

Reply:
> I don't think this naming has any impact on whether this class is loaded, based on the discussion in this thread [#41857 (comment)](https://github.com/ceph/ceph/pull/41857#discussion_r652276366) .

True. I was under the impression that all classes and methods with names starting with "Test" would be loaded but that's not the case.

> I think the right thing to do here is multiple inheritance. This class should not inherit `CephFSTestCase`. Instead, child classes should inherit `TestAdminCommands` and `CephFSTestCase`.

I don't see the need for multiple inheritance. Conceptually, it probably is more apt but practically inheriting 2 classes in every class in test_admin.py is redundant effort.